libraries/GCS_MAVLink/GCS.h

```diff
@@ -648,6 +648,9 @@ class GCS
 
     // get the VFR_HUD throttle
     int16_t get_hud_throttle(void) const { return num_gcs()>0?chan(0).vfr_hud_throttle():0; }
+
+    // method to allocate _statustext_sem
```

Comment:
A say-nothing-interesting comment which is going to get out of date.

Reply:
fixed